std/parallelism.d

```diff
@@ -3897,6 +3971,14 @@ version(unittest)
 // These are the tests that should be run every time Phobos is compiled.
 unittest
 {
+    import std.algorithm.iteration : filter, map, reduce;
+    import std.algorithm.comparison : equal, max, min;
```

Comment:
Swap these two imports for alphabetical order, can you make sure the rest are in alphabetical order also?